openapi3filter/validation_error_test.go

```diff
@@ -372,15 +372,6 @@ func getValidationTests(t *testing.T) []*validationTest {
 				Title:  `property "name" is missing`,
 				Source: &ValidationErrorSource{Pointer: "/category/tags/0/name"}},
 		},
-		{
```

Comment:
can test both read/writeonly in openapi3filter/validate_readonly_writeonly_test.go instead

Reply:
Sounds good to me.